frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx

```diff
@@ -10,12 +10,12 @@ import React, { Component } from "react";
 import ReactDOM from "react-dom";
 import _ from "underscore";
 
-import { waitTimeContext } from "metabase/context/wait-time";
 import CS from "metabase/css/core/index.css";
 import { isCypressActive } from "metabase/env";
+import { delay } from "metabase/lib/delay";
 import resizeObserver from "metabase/lib/resize-observer";
 
-const WAIT_TIME = 300;
+const WAIT_TIME = delay(300);
```

Comment:
2 things here:
1. you should also remove anything mentioning `context` in this file at lines 83-85 for example.
2. delay 0 vs refreshMode = 'none' are slightly different, I think it renders 1 event loop apart, I'm not sure of the implication of that, but I don't think it should make much difference.  Let's keep an eye on it.